### ISSUE TITLE
Loading Crash on V1.10.12

### ISSUE THREAD

**Issue by CaptainCJH** [anegostudios/VintageStory-Issues#338]
**Game Version:** 1.10.12
**Platform:** Windows 10
**Modded:** No

### Description
Game crashed upon attempting to load a new survival game after updating to new version.

### How to reproduce
Could not reproduce.

### Expected behavior
To load the game.

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.10.12 (Stable)9/24/2019 3:00:16 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.HudElementBlockInteractionHelp.OnBlockChanged(BlockPos pos, Block oldBlock)
   at Vintagestory.API.Client.BlockChangedDelegate.Invoke(BlockPos pos, Block oldBlock)
   at _7zeZUcEGU2bnlcmDU2snMtGxGSe._LM1gE8Sb5XjCBTEUoVEe76GfIlq(_YQvK0FsJHSuLbtBLFq7kjfAQeKf , BlockPos , Block )
   at Vintagestory.Client.NoObf.SystemNetworkProcess.<>c__DisplayClass18_1.<ProcessInBackground>b__1()
   at _YQvK0FsJHSuLbtBLFq7kjfAQeKf._nSMHJbDcCYQD419vxR4b0DSemjx(Single )
   at _BZvuKipvjgOcnAyEtV5K0Cs4IlG._mQeczwMSzvMtKMADNtmDqxQ3n5e(Single )
   at _FH6hvsknNaImtXLoDARCT1Kxyfi._oQSULXPyQlh9mb7cqzSfhFJKIP0(Single )
   at _FH6hvsknNaImtXLoDARCT1Kxyfi._tKKbzJkc7DUhKi6uwBKBfaFTWHX(Single )
   at _jm4dFkqFzcufJfGE0GepdiZBsjJA._qNgPco5xeAnQYRQfv0ddffm4ptf(Object , FrameEventArgs )
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _SNpm6CX5gDSCaLxDMWSDr5aLrr8._NFfXgYz59dCAY8ZgF8EdndepnJv(_nTDv2rJfEPo2JjsLUmdGC7Wccck , String[] )
   at _Qqios9QvZpcSUwRbGSu9gbBPcKC._NFfXgYz59dCAY8ZgF8EdndepnJv(ThreadStart )

**Comment by copygirl** [comment]
Having the same or very similar crash here:
```
Version: v1.10.13 (Stable)9/25/2019 4:04:30 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.HudElementBlockInteractionHelp.OnBlockChanged (Vintagestory.API.MathTools.BlockPos pos, Vintagestory.API.Common.Block oldBlock) [0x00010] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at (wrapper delegate-invoke) <Module>.invoke_void_BlockPos_Block(Vintagestory.API.MathTools.BlockPos,Vintagestory.API.Common.Block)
  at _g1UDXPx7SOxPV5BQisA6g8SeCzb._LR2dNYTAnaz0aAfz5FPtsQy0HhF (Vintagestory.API.MathTools.BlockPos , Vintagestory.API.Common.Block ) [0x0000a] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at _DfP5BPZ3xXnIZ8NJ9s2CkYs7QHf._LR2dNYTAnaz0aAfz5FPtsQy0HhF (_YZBE4WcPzfzBp4bXTsW7WW83gLk , Vintagestory.API.MathTools.BlockPos , Vintagestory.API.Common.Block ) [0x00000] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at Vintagestory.Client.NoObf.SystemNetworkProcess+<>c__DisplayClass18_1.<ProcessInBackground>b__1 () [0x00041] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at _YZBE4WcPzfzBp4bXTsW7WW83gLk._ydkDlS5eK4M6hLrgELnc7zsGWkp (System.Single ) [0x00158] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at _Wv5qhHoa47S9TRXsRJHsvjIrFdn._n0Tu5UCHKZPQzPuX3yhmQQ2iPIC (System.Single ) [0x00401] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at _JSVc85euyPXYzGpbqO9wGnpYLid._zbeUi6VsGOarUk0v58TUSvqAzMq (System.Single ) [0x0017b] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at _JSVc85euyPXYzGpbqO9wGnpYLid._khjvrzsEg9EgAdOt0AWFYCdluTc (System.Single ) [0x000c2] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
  at _iSC3VjqyYpzxo1lqPddDAocByHB._iQRGuFIBg2qRhWwCx6ej7S1xS0o (System.Object , OpenTK.FrameEventArgs ) [0x000c3] in <e590f2179c7e402abc2ddaf0f4e656f7>:0 
```
Unfortunately, I tried with a brand new world in 1.10.13 and it just crashes repeatedly.

**Comment by tyronx** [comment]
thanks, should be fixed in v1.10.14, releasing in an hour or so
